packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -269,32 +269,11 @@ const App = (props) => {
         history.push(path)
     }
 
-    // A major pain point for the current development of Storefront Preview is that we don't have
-    // static URLs on deployed sites. The path /mobify/bundle/123/static/storefront-preview.js
-    // changes to a different number with every deployment. Once we figure out the static URL we can
-    // update it for production, but will still need to come up with something for staging...
-
-    const storefrontPreviewClientScript =
-        process.env.NODE_ENV === 'development'
-            ? 'http://localhost:4000/mobify/bundle/development/static/storefront-preview.js'
-            : // Gotta update this manually for now... :(
-              // : 'https://runtime.commercecloud.com/storefront-preview.js'
-              null
-
-    if (!storefrontPreviewClientScript) {
-        // Deliberately aggressive because this branch exists solely for Storefront Preview development
-        return (
-            <Box className="sf-app">
-                Hey! You forgot to set a Storefront Preview client script URL!
-            </Box>
-        )
-    }
+    const [, forceUpdate] = useReducer((x) => x + 1, 0)
```

Comment:
This will run either in Preview or non-Preview mode? We only want to run this when storefront in Preview Mode.
Can we detect when it is actually in Preview Mode only? we need this line of code to forceUpdate, what if users delete this line of code? Preview would be broken. Can we move it inside the react-sdk instead? 🤔